.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,23 @@
+name: Release Drafter
```

Comment:
did you mean to include this file? All the other PR's are just the release-drafter config file, and not the workflow

Reply:
Removed.  I opened a few PRs before remembering the organizational-level approach.